pkg/server/router/credential_test.go

```diff
@@ -275,4 +275,72 @@ func TestCredentialRouter(t *testing.T) {
 		assert.NotEmpty(tt, credStatusMapTwo["statusListIndex"])
 
 	})
+
+	t.Run("Credential Status List Test On Create", func(tt *testing.T) {
```

Comment:
It's unclear what **behavior** this test is exercising. Would recommend following this [naming best practices](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests) for this test as well. Perhaps worth also splitting it up.